mappings/dataprepper/AWS/v1.1.0/VPC Flow/samples/vpcflowlog.ocsf

```diff
@@ -0,0 +1,69 @@
+{
+	"cloud": {
+		"account_uid": "987654321098",
+		"region": "us-west-2",
+		"zone": "use2-az2",
+		"provider": "AWS"
+	},
+	"action": "Allowed",
+	"action_id": 1,
+	"status_code": "OK",
+	"traffic": {
+		"bytes": 85,
+		"packets": 10
+	},
+	"src_endpoint": {
+		"ip": "192.168.1.10",
+		"port": 8080,
+		"svc_name": "amazon-s3",
+		"subnet_uid": "subnet-33333333333333333",
+		"vpc_uid": "vpc-44444444444444444"
+	},
+	"dst_endpoint": {
+		"ip": "192.168.1.20",
+		"port": 443,
+		"svc_name": "amazon-ec2",
+		"interface_uid": "eni-22222222222222222",
+		"instance_uid": "i-111111111111111111"
+	},
+	"connection_info": {
+		"protocol_num": 17,
+		"protocol_ver": "IPv6",
+		"tcp_flags": 6,
+		"direction": "egress",
+		"direction_id": 2,
+		"boundary_id": 99,
+		"boundary": "vpn",
+		"start_time": 1653200123,
+		"end_time": 1653200100
+	},
+	"time": 1653200100,
+	"type_name": "Network Activity: Traffic",
+	"type_uid": 400105,
+	"activity_id": 5,
+	"activity_name": "Traffic",
+	"class_uid": 4001,
+	"category_uid": 4,
+	"category_name": "Network Activity",
+	"metadata": {
+		"product": {
+			"name": "Amazon VPC",
+			"feature": {
+				"name": "Flowlogs"
+			},
+			"vendor_name": "AWS"
+		},
+		"profiles": [
+			"cloud",
+			"security_control"
+		],
+		"version": "1.1.0"
+	},
+	"severity_id": 1,
+	"severity": "Informational",
+	"disposition": "Allowed",
+	"pkt_src_aws_service": "amazon-s3",
+	"pkt_dst_aws_service": "amazon-ec2",
+	"sublocation_type": "subnet",
+	"sublocation_id": "subnet-33333333333333333"
+}
```

Comment:
Curious, did you not want to fill in `status/status_id`? It does seem a but redundant w/`action` and `activity` though it is `recommended`.

Reply:
Similar for `class_name` - it's optional but since `category_name` and `type_name` are populated, seems like it would be consistent.  For `status/status_d` it is usually `Success/1` unless there is an actual error of the service itself (not just `REJECT` for example).  I'm not crazy about our convention for `status` but this is how Roumen had explained it, and I went along with it :-).  `status_code` in this case would be interesting as to what its values might be from the source.